styles/style.css

```diff
@@ -0,0 +1,117 @@
+/* 
+Dark Red: #8F0C0C
+Rosewood: #5F1212 
+Dark Sienna: #230707
+Tuscany: #CAA2A2
+Snow: #F7F2F2
+*/
+
+* {
+    margin: 0;
+    padding: 0;
+}
```

Comment:
Tip: You seem to have a lot of styles repeated here and in `index_style.css`.  

Since you can include both sheets on pages you can DRY up your code by removing redundant rules.